README.md

```diff
@@ -20,44 +20,59 @@ The main services to request information are:
 * Information about lines
 
 OJP systems interacting with each other additionally use:
-* Getting information about Exchange Points (places when a Trip switches from one OJP system to another)
-* Using a multi point trip service (n origins and m destinations)
-
+* Getting information about Exchange Points (places when a trip switches from one OJP system to another)
+* Using a multipoint trip service (n origins and m destinations)
 
 ## Standard document
 OJP is a CEN standard. Obtain a copy of the standard document
 ["CEN/TS 17118:2017: "Intelligent transport systems - Public transport - Open API for distributed journey planning"](https://standards.cen.eu/dyn/www/f?p=204:110:0::::FSP_PROJECT:62236&cs=1985DBD613F25D179FB65A73B0FDA4DB7)
 at [CEN](https://www.cen.eu).
 
-OJP is part of the [Transmodel](https://www.transmodel-cen.eu/)) family of standards together with [NeTEx](https://netex-cen.eu/), [SIRI](https://www.transmodel-cen.eu/siri-standard/)  and OPRA. It also contains elements from [DATEX II](https://datex2.eu/).
+OJP is part of the [Transmodel](https://www.transmodel-cen.eu/) family of standards together with
+[NeTEx](https://netex-cen.eu/), [SIRI](https://www.transmodel-cen.eu/siri-standard/) and
+[OpRa](https://www.opra-cen.eu/). It also contains elements from [DATEX II](https://datex2.eu/).
```

Comment:
I am not a fan of what happened here. Keep it a single line.

Reply:
The point is that every time someone needs to change the line, we are mixing formatting with code. Keep these thing a single line, and let your text editor do the wrapping, if that is required.

Reply:
I disagree, but it's not important enough to me that it should block this pull request.
https://about.gitlab.com/blog/2016/10/11/wrapping-text/